### PR TITLE
Cfg: add wstETH on BSC configs

### PR DESCRIPTION
**Pull request by TheDZhon**
- [x] Add [mainnet](
https://research.lido.fi/t/wormhole-x-axelar-lido-bridge-implementation-for-wsteth-on-bnb-chain/6012/31) configs
- [x] Add [testnet](https://research.lido.fi/t/wormhole-x-axelar-lido-bridge-implementation-for-wsteth-on-bnb-chain/6012/32) configs

---

Axelar sources are flattened with Hardhat 2.22.6 so it's impossible to automate the checks at the moment :dagger:  

- Transceiver Structs (used by Axelar Transceiver, Ethreum): [0xa12bc993d8144404a8c8c812816048275a066ced](https://etherscan.io/address/0xa12bc993d8144404a8c8c812816048275a066ced)
- Transceiver Structs (used by Axelar Transceiver, BSC): [0x27a3daf3b243104e9b0afae6b56026a416b852c9](https://bscscan.com/address/0x27a3daf3b243104e9b0afae6b56026a416b852c9)
- Axelar Transceiver (Ethereum): [0x723AEAD29acee7E9281C32D11eA4ed0070c41B13](https://etherscan.io/address/0x723AEAD29acee7E9281C32D11eA4ed0070c41B13) (proxy)
- Axelar Transceiver (Ethereum): [0x87fc4b27385bb4e69a40027931229d74ef4d1943](https://etherscan.io/address/0x87fc4b27385bb4e69a40027931229d74ef4d1943) (impl)
- Axelar Transceiver (BSC): [0x723AEAD29acee7E9281C32D11eA4ed0070c41B13](https://bscscan.com/address/0x723AEAD29acee7E9281C32D11eA4ed0070c41B13) (proxy)
- Axelar Transceiver (BSC): [0xa1ebb6a4b856df8bf6c3aca88a9115a9ab3b2e02](https://bscscan.com/address/0xa1ebb6a4b856df8bf6c3aca88a9115a9ab3b2e02) (impl)

### CODE REVIEW

File: config_samples/bsc/mainnet/wsteth_mainnet_config_binance.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "contracts": {
+    "0x6981F5621691CBfE3DdD524dE71076b79F0A0278": "ERC1967Proxy",
+    "0xe82c2a5846cfb6d8683d6b636719e7aa61486838": "NttManager",
+    "0xbe3F7e06872E0dF6CD7FF35B7aa4Bb1446DC9986": "ERC1967Proxy",
+    "0xb948a93827d68a82F6513Ad178964Da487fe2BD9": "WormholeTransceiver",
```
Comment (tamtamchik) [inline config_samples/bsc/mainnet/wsteth_mainnet_config_binance.json:6]: It clashes with the ETH NTT Manager proxy, but I guess there is nothing we can do...

**Review by tamtamchik — APPROVED** [review]
LGTM 👍 

Some suggestions in separate PR to this branch: https://github.com/lidofinance/diffyscan/pull/56

File: config_samples/bsc/testnet/wsteth_testnet_config_ethereum.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "contracts": {
+    "0x8B715EAf61A7DdF61C67d5D46687c796D1f47146": "ERC1967Proxy",
+    "0x607b139bfee21b2676ee664a237a70d737b9466e": "NttManager",
+    "0xF2bc73502283fcaC4b047dfE45366d8744daaC5B": "ERC1967Proxy",
+    "0xaf99eeb712511303f8b829707aafc5881a125771": "WormholeTransceiver",
```
Comment (tamtamchik) [inline config_samples/bsc/testnet/wsteth_testnet_config_ethereum.json:6]: ❗ On the testnet WormholeTransceiver.sol, Transceiver.sol, ITransceiver.sol have different code.

File: config_samples/bsc/mainnet/wsteth_mainnet_config_binance.json (hunk 1 above)
Comment (TheDZhon) [reply config_samples/bsc/mainnet/wsteth_mainnet_config_binance.json:6]: yeah, the thing sucks

File: config_samples/bsc/testnet/wsteth_testnet_config_ethereum.json (hunk 2 above)
Comment (TheDZhon) [reply config_samples/bsc/testnet/wsteth_testnet_config_ethereum.json:6]: Yeah, got catch, extra getters :-1:

**Pull request merged**